docs/governance-strategy.md

```diff
@@ -0,0 +1,137 @@
+   ```
+   ## Context
+   
+   provide context needed to understand this proposal. Describe the problem this proposal will solve.
+   
+   ## Proposal
+   
+   describe your proposal in informal but specific terms.
+   
+   ## Impact
+   
+   describe the implications of this proposal and the impact it will have.
+   ```
+2. **Discuss and incorporate feedback** with others on the team. If there are objections or suggestions, do your best to incorporate them into your proposal.
+3. **Make a Pull Request** to the `Team Compass` repository (or another location if appropriate) and link it to the issue. This is the "formal change" that you wish to make.
+4. **Make a decision**. Steering Council members may approve PRs to change policy. To approve a change, use the "Approve" feature in the GitHub UI. To request blocking changes, use the "Request Changes" feature in the GitHub UI[^blocking]. PRs to change organizational policy may be merged when the following conditions are met:
+   - Have been open for five working days
+   - Have `Approval` from at least **One  Steering Council member**
+   - Have no `Request Changes` from a Steering Council member.
+   - If there are unresolveable objections from a Steering Council member, a decision to merge is made with a majority vote.
+   - If all steering council members `Approve`, it may be merged any time.
+
+
+[^blocking]: When blocking any change or objecting to a proposal, provide a rationale for what must be changed and why you believe it is critically important. _Do not disapprove because of differences in opinion. Only disapprove if you have a major strategic concern_. See [Strategies for integrating objections](https://www.sociocracyforall.org/strategies-for-integrating-objections/) for what we are aiming for.
+
+
+### `executablebooks/myst-spec`: The MyST Specification
+
+`executablebooks/myst-spec` is the Source of Truth for the MyST Specification. It uses a combination of documentation, examples, schemas, tests, etc to define MyST markdown in a form that others could use to implement parsers and renderers. It is versioned and includes "releases" in order to make it easy for downstream implementors to track the changes they need to make.
+
+```{admonition} Implementation detail
+This repository will need to be modified as-needed to be the source of truth for the MyST Specification, and to have enough information to teach newcomers about its structure and function. One example for inspiration: [the Zarr specifications page](https://zarr.readthedocs.io/en/stable/spec.html).
+```
+
+The MyST Specification is changed through the MyST Enhancement Proposal process, described below.
+
+### `executablebooks/myst-eps`: MyST Enhancement proposals
+
+Because the MyST specification has many interior and exterior stakeholders, we use a more formal and structured process around changing the specification. This process is encoded in `executablebooks/myst-eps` and described briefly below
```

Comment:
Is this an SoT? It doesn't read like the others, which start by noting what they're a Source of Truth of. 

Based on my current read, it feels like there are two SoTs, and the _MyST Enhancement proposals_ section should be nested one more level (####) and its title aligned with _How to change team policy_ (e.g., _How to change the MysT Specification: MyST Enhancement proposals_)

Reply:
The second sentence on line 85 is missing a period at the end.

Reply:
Ha, that's how it was before and I changed it at the the last minute because I have a personal aversion to four-deep nested headers 😅, but it sounds like I should not have un-nested it after all. I'll change it back and I agree w/ you!